tests/chainer_tests/training_tests/triggers_tests/test_interval_trigger.py

```diff
@@ -41,6 +47,13 @@ def _test_trigger(self, updater, trigger, expecteds):
         self.assertEqual(trigger(trainer), expected)
 
 
+def _test_trigger_stop(self, updater, trigger, expecteds):
```

Comment:
If I understand correctly, this function is for stop triggers. If so, could you change the function name to make its role clearer.